packages/circuit-ui/components/Checkbox/Checkbox.tsx

```diff
@@ -250,10 +251,7 @@ export const Checkbox = forwardRef(
           disabled={disabled}
           invalid={invalid}
           ref={ref}
-          // @ts-expect-error Change is handled by onClick for browser support, see https://stackoverflow.com/a/5575369
-          onClick={handleChange}
-          // Noop to silence React warning: https://github.com/facebook/react/issues/3070#issuecomment-73311114
-          onChange={() => {}}
+          onChange={handleChange}
```

Comment:
Would this still be ways to make this work in IE if necessary? (e.g. with a polyfill/workaround)

Reply:
Not sure. The limitation here is:

> Internet Explorer only fires the `onchange` event when the checkbox loses focus (`onblur`).

So it still works, just not well.